machinist/mserver/mserver.go

```diff
@@ -0,0 +1,47 @@
+package mserver
+
+import (
+	"github.com/enfabrica/enkit/machinist"
+	machinist_rpc "github.com/enfabrica/enkit/machinist/rpc/machinist"
+	"google.golang.org/grpc"
+	"sync"
+)
+
+func New(mods ...Modifier) (*server, error) {
+	s := &server{
+		killChannel: make(chan error),
+		Controller:  &Controller{},
+		SharedFlags: &machinist.SharedFlags{},
+	}
+	for _, m := range mods {
+		if err := m(s); err != nil {
+			return nil, err
+		}
+	}
+	return s, nil
+}
+
+type server struct {
+	insecure      bool
+	runningServer *grpc.Server
+	killChannel   chan error
+	Controller    *Controller
+	sync.Mutex
+	*machinist.SharedFlags
+}
+
+func (s *server) Flags() *machinist.SharedFlags {
+	return s.SharedFlags
+}
+
+func (s *server) Start() error {
```

Comment:
same comment as another PR. Start() generally starts something and returns, Run() or Serve() block. This blocks, so I would rename it.

Reply:
Changed to run